js/bitopro.js

```diff
@@ -950,6 +951,29 @@ module.exports = class bitopro extends Exchange {
         return this.parseOrder (response, market);
     }
 
+    async cancelOrders (ids, symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelOrders() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const id = market['id'].toUpperCase ();
+        const request = {};
+        request[id] = ids;
+        const response = await this.privatePutOrders (this.extend (request, params));
```

Comment:
I think `privatePutOrders` is used for order-editing (changing the amounts or prices of the open orders), and for canceling we need to use the `delete` methods?

Reply:
Yes, if they follow the REST api specification (which they didn't). I felt weird to use put......

Reply:
What about this endpoint is it the same functionality does it work? 

- https://github.com/bitoex/bitopro-offical-api-docs/blob/master/v3-1/rest-1/auth/cancel-all.md

Reply:
Actually, you're right, nevermind.